UnityProject/Assets/Scripts/Shuttles/ShuttleHeater.cs

```diff
@@ -9,4 +9,14 @@ public override void SpriteChange()
 	{
 		return;
 	}
+
+	public override void SetSpriteLayer()
+	{
+		return;
```

Comment:
why even override it then?

Reply:
to cancel it and not call the base.SetSpriteLayer(), is there a more elegant way to do it?